scripts/export_models_ops_correlation.py

```diff
@@ -0,0 +1,293 @@
+    for col in range(1, sheet.max_column + 1):
+        sheet.cell(row=1, column=col).fill = blue_fill
+        sheet.cell(row=1, column=col).border = thin_border
+        sheet.cell(row=1, column=col).alignment = center_aligned
+
+    # Fill last column(i.e OpCount) with red color
+    for rol in range(2, sheet.max_row + 1):
+        sheet.cell(row=rol, column=sheet.max_column).fill = red_fill
+
+    # Set column width for cells
+    column_offset = 2
+    for col in ws.columns:
+        column = col[0].column_letter
+        ws.column_dimensions[column].width = max_model_name_len + column_offset
+
+    # Loop over the cells in the sheet and apply color based on value
+    for row in sheet.iter_rows(min_row=2, max_row=sheet.max_row, min_col=2, max_col=sheet.max_column):
+        for cell in row:
+            cell.border = thin_border
+            cell.alignment = center_aligned
+            if cell.value == "Y":
+                cell.fill = green_fill
+            elif cell.value == "N":
+                cell.fill = black_fill
+
+    # Save the Excel file with formatting
+    writer.save()
+    writer.close()
+
+    logger.info(
+        f"Saved cross correlation details between model variants vs unique op configs to {os.path.join(output_directory_path,output_file_name+'.xlsx')}"
+    )
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(
+        description="Gather cross correlation details between model variants vs unique op configs"
+    )
+    parser.add_argument(
+        "-c",
+        "--compile_depth",
+        choices=["GENERATE_INITIAL_GRAPH", "PRE_LOWERING_PASS"],
+        required=True,
+        help="Choose compilation depth for extracting ops configuration for the all the models present in pytest_directory_path",
+    )
+    parser.add_argument(
+        "-i",
+        "--pytest_directory_path",
+        required=True,
+        help="Specify the directory path containing models test",
+    )
+    parser.add_argument(
+        "-f",
+        "--outputfilename",
+        default="Models_Ops_cross_correlation_data",
+        required=False,
+        help="Specify the output file name for xlsx/csv file",
+    )
+    parser.add_argument(
+        "-o",
+        "--output_directory_path",
+        default=os.getcwd(),
+        required=False,
+        help="Specify the output directory path for saving the xlsx/csv file",
+    )
+    parser.add_argument(
+        "-s",
+        "--do_save_xlsx",
+        default=True,
+        required=False,
+        help="Specify the output directory path for saving the xlsx/csv file",
+    )
+    args = parser.parse_args()
+
+    compilation_depth = args.compile_depth
+    pytest_directory_path = args.pytest_directory_path
+    output_file_name = args.outputfilename
+    output_directory_path = args.output_directory_path
+    do_save_xlsx = args.do_save_xlsx
+
+    run_and_export_models_unique_ops_configuration(compilation_depth, pytest_directory_path)
```

Comment:
Let's add a doc section on how to use this script as well :))

For example here:
- https://docs.tenstorrent.com/tt-forge-fe/tools.html

Reply:
Sure I will add it in doc

Reply:
Need to update the doc

Reply:
Updated the doc.